spec/mandrill_dm/message_spec.rb

```diff
@@ -131,6 +131,16 @@ def new_mail(options = {}, &blk)
       expect(message.global_merge_vars).to eq(global_merge_vars)
     end
 
+    it 'does not use gsub to convert string into JSON' do
```

Comment:
> Perhaps retrieves/restores the hash to original state captures the idea?

This test is only against 1 way of converting string into JSON - replacing `:keys => ` by `"keys" : ` Other methods may pass it (but will break on something else, e.g. commas).